app/services/UserService.php

```diff
@@ -16,11 +18,39 @@ public function createUser(array $data)
     }
 
     /**
-     * @param array $data
+     * @param string $email
      * @return mixed
      */
-    public function checkUser(array $data)
+    public function getUserByEmail(string $email)
```

Comment:
let's show return values type (for each method in service)
in. this case it. will be:
public function getUserByEmail(string $email): User - because method returns instance of User class